install-dev/upgrade/sql/1.7.6.0.sql

```diff
@@ -50,3 +50,207 @@ INSERT IGNORE INTO `PREFIX_hook` (`id_hook`, `name`, `title`, `description`, `po
   (NULL, 'actionBuildMailLayoutVariables', 'Build the variables used in email layout rendering', 'This hook allows to change the variables used when an email layout is rendered', '1'),
   (NULL, 'actionGetMailLayoutTransformations', 'Define the transformation to apply on layout', 'This hook allows to add/remove TransformationInterface used to generate an email layout', '1')
 ;
+INSERT IGNORE INTO `PREFIX_hook` (`id_hook`, `name`, `title`, `description`, `position`) VALUES
+(NULL,"actionSqlRequestFormBuilderModifier","Modify sql request identifiable object form","This hook allows to modify sql request identifiable object forms content by modifying form builder data or FormBuilder itself","1"),
```

Comment:
Maybe clean the sql like above :roll_eyes:

Reply:
Single quotes you mean ?

Reply:
yep and spaces after comma :heart:

Reply:
Why the spaces ? For readability ?
It's a 200 lines SQL query 😬 I dont know if readability is relevant here

Reply:
By the way maybe we could avoid these big SQL statements by writing a dedicated script for adding the new hooks 🤔 . Because we're going to have a new big batch next version, and probably the one after it too...

Reply:
It's always consistent to have the same way to write anything, I can do it for you :heart_eyes: